spec/api-native-image-spec.js

```diff
@@ -186,7 +192,8 @@ describe('nativeImage module', () => {
     });
   });
 
-  describe('createFromBuffer(buffer, options)', () => {
+  // TODO fix nativeImage.createFromBuffer on WOA
+  ifdescribe(process.platform !== 'win32' || process.arch !== 'arm64')('createFromBuffer(buffer, options)', () => {
```

Comment:
nit:
```suggestion
  ifdescribe(!(process.platform === 'win32' && process.arch === 'arm64'))('createFromBuffer(buffer, options)', () => {
```

is a easier to read imo!

Reply:
👍